contribs/gnogenesis/internal/txs/txs_add_packages_test.go

```diff
@@ -60,66 +63,236 @@ func TestGenesis_Txs_Add_Packages(t *testing.T) {
 		assert.ErrorContains(t, cmdErr, errInvalidPackageDir.Error())
 	})
 
-	t.Run("invalid deployer address", func(t *testing.T) {
+	t.Run("nonExistentKeyName", func(t *testing.T) {
```

Comment:
Please fix up this test name

Reply:
Same for the other test cases

Reply:
Fixed :)
09dce4322f68e8f7aa366c3c33da287d5761d8ae